services/api/src/models/user.js

```diff
@@ -15,14 +16,16 @@ const schema = new mongoose.Schema(
         type: String
       }
     ],
-    name: { type: String, trim: true },
-    hashedPassword: { type: String },
-    deletedAt: { type: Date },
+    name: {
+      type: String,
+      trim: true
+    },
+    hashedPassword: {
+      type: String,
+      private: true
```

Comment:
Let's make sure we have a check in the unit tests for `hashedPassword` not showing up through the API through autopopulate

Reply:
just added!